core/sql/optimizer/EncodedKeyValue.cpp

```diff
@@ -68,14 +68,22 @@ NAString * getMinMaxValue(TrafDesc * column,
     return NULL;
   
   Lng32 buflen = type->getTotalSize();
+  Lng32 nullHdrSize = 0;
   char * buf = new char[buflen]; // deleted at the end of this method
-
+  if (type->supportsSQLnullPhysical())
+  {
+      nullHdrSize = type->getSQLnullHdrSize();
+      for(int i = 0; i < nullHdrSize; i++)
```

Comment:
Not familiar with this part, but would it be more efficient to use memset() to do the initial work rather than using a for loop? Just my opinion...